.insomnia/Request/req_2b193752cad6487691cd5ce0aef044d5.yml

```diff
@@ -0,0 +1,87 @@
+_id: req_2b193752cad6487691cd5ce0aef044d5
+type: Request
+parentId: fld_80bc7ee1b14547dc860545b5af4878c5
+modified: 1687182853229
+created: 1687182078953
+url: "{{ _.base_url }}/api/v1/reviews/{{ _.review_id }}/customize_by_filter"
+name: Customize by filter
+description: >
+  Thiss endpoint takes a filter parameters which apply it to get certain
+  review's articles and apply a certain customization on it.
+
+
+  All the customization are done in a background job, so the return status will be `ok` which means that the customization job was enqueued successfully.
+
+
+  Params:
+
+
+  - `extra`: This is a dictionary that contains facet-based filter parameters, which are described below. Parameters ending with `[]` can contain multiple values.
```

Comment:
If this is a copy, just refer to the other endpoint.

Reply:
I prefer to document it in the same endpoint as it's not an exact copy.

Reply:
Done.